src/main/scala/ru/org/codingteam/horta/protocol/jabber/MucMessageHandler.scala

```diff
@@ -102,9 +102,9 @@ class MucMessageHandler(locale: LocaleDefinition,
       case None =>
         log.warning(s"Cannot find participant $jid in the participant list")
         CommonAccess
-      case Some(participant) => participant.affiliation match {
+      case Some(p) => p.affiliation match {
```

Comment:
If shortening variables' names really helps to fix such errors I approve this shit.